apps/app/pages/dashboard/election-explorer/candidates/index.tsx

```diff
@@ -7,15 +7,16 @@ import { useTranslation } from "@hooks/useTranslation";
 import { get } from "@lib/api";
 import { withi18n } from "@lib/decorators";
 import type { Page } from "@lib/types";
-import { GetStaticProps, InferGetStaticPropsType } from "next";
+import { isEmpty } from "lodash";
```

Comment:
Better to use the single file export with lodash. Refer here: https://github.com/vercel/next.js/issues/12557

```ts
import isEmpty from 'lodash/isEmpty'`
```

Nevertheless, I should tell, that I am indecisive on this issue, which is why, the barrel file, `components/index` file still exists. Its convenient to have it around, but it can bloat our build files. Importing individual files is more efficient but can get tedious if you have lots of them.

So for now, keep to importing individual files, especially for 3rd party libs.